apps/hannk/halide/CMakeLists.txt

```diff
@@ -8,49 +8,49 @@ set(CMAKE_CXX_EXTENSIONS NO)
 
 # ----------------------------
 
-if (NOT CMAKE_CROSSCOMPILING)
+add_custom_target(hannk_tools)
```

Comment:
Yeah, in #6366 I chose `${PROJECT_NAME}::generators::` as a convention. We could stick `halide_` in there somewhere, but it felt wordy to me :shrug:

Reply:
Since CMake (and Python, and... etc) use "generator" to mean very different things from Halide, I'd vote for the clarification. (These names aren't really 'seen' by the user unless they read the CMake files anyway, so maybe I'm overthinking it, but long names seem cheap here.)

Reply:
I renamed it from `hannk_tools` to `hannk-halide_generators`.